src/index.ts

```diff
@@ -12,3 +12,7 @@ import { Constants } from 'stanza';
 const JingleReasonCondition = Constants.JingleReasonCondition;
 
 export { utils, mediaUtils, JingleReason, JingleReasonCondition, JingleInfo };
+
+/* make sure to export the default */
+import GenesysCloudWebrtSdk from './client';
+export default GenesysCloudWebrtSdk;
```

Comment:
Since we are exporting the barrel file now, we need to explicitly export this class as the default to keep backwards compatibility.